src/test/inputs/common.spec.scss

```diff
@@ -0,0 +1,105 @@
+@import 'true';
+@import '../../components/inputs/common';
+
+@include describe('Input Common (shared input mixins)') {
+  @include it('Outputs Vanilla field base properties') {
+    @include assert {
+      @include output {
+        @include vanilla-field-base();
+      }
+
+      @include expect {
```

Comment:
This is nice, but i'm not sure that we really need to test all rules. It seems like a lot of work to maintain in the long run.

Maybe it would suffice to check that the mixin is found by simply:
```scss
@include expect {
  @include vanilla-field-base();
}
```

Then the test should fail if the expected mixin is not found in the imported file. What do you think?

Reply:
Ok, that actually does the job! I didn't realise.. But sure - the whole point of the tests is to make sure all imports are working after refactoring, and your proposal should do just that.